src/docs/product/alerts/index.mdx

```diff
@@ -44,6 +44,10 @@ You can find a full list of available metric alerts in [Metric Alerts](/product/
 
 When you create a new project in [sentry.io](https://sentry.io), you can select a default issue alert. However, you can also [create your own alerts](/product/alerts/create-alerts/) to suit your team’s needs, using these [best practices](/product/alerts/best-practices/) as a guide.
 
+## Muting Alerts
+
+Issue alerts can be muted on the alert rule details page by clicking the "Mute" button. This will mute the alert from notifying you until you click "Unmute". If you want to mute the alert from firing entirely, select "Mute for everyone" from the dropdown.
```

Comment:
maybe mention that if you unmute then it will first try to unmute the one for all then the one for you. seems like an edge case but might be good to have that written down

Reply:
Hmm this feels like we should have a visual indication of whether it's muted for just you or everyone

Reply:
we have a visual indicator but looking at the code (https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_rule_details.py#LL97C36-L97C36) we don't give priority to everyone/just for yourself when determining the banner to show. the endpoint code for deleting does delete both right now (everyone/yourself) if they exist

Reply:
Oh somehow I totally missed the banner. Ehh I don't think it's necessary since the only way to get into a state where you'd be worried about what would happen would be if you used the API, right? Like if you muted the same alert for yourself and everyone and then clicked unmute?

Reply:
yeah that makes sense.